blueprints/app/files/package.json

```diff
@@ -29,11 +29,11 @@
     "ember-cli-shims": "^1.1.0",
     "ember-cli-sri": "^2.1.0",
     "ember-cli-uglify": "^1.2.0",
-    "ember-data": "^2.13.2",
+    "ember-data": "~2.14.0",
```

Comment:
I think this should be `2.14.3` just for safesies? The previous 2.14 releases have some problems.